common/src/main/resources/Operator.properties

```diff
@@ -325,4 +325,7 @@ WLSWH-0020=Change request to domain resource ''{0}'' cannot be honored because t
 WLSWH-0021=Change request to domain resource ''{0}'' causes the replica count of each cluster in ''{1}'' to exceed its cluster \
   size ''{2}'' respectively
 WLSWH-0022=Scale request to cluster resource ''{0}'' cannot be honored because the replica count would exceed the \
-  cluster size ''{1}''
\ No newline at end of file
+  cluster size ''{1}''
+WLSWH-0023=Unsupported introspector environment variable(s) ''{0}'' defined in ''introspector.serverPod.env'', \
+  supported values are ''JAVA_OPTIONS'', ''USER_MEM_ARGS'', ''NODEMGR_JAVA_OPTIONS'' and ''NODEMGR_MEM_ARGS''
```

Comment:
Minor: Consider passing in the list as I assume this is coded up in a central place in Java.

Reply:
Changed.